apps-rendering/src/components/InlineSkipToWrapper/index.tsx

```diff
@@ -0,0 +1,78 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import { border, neutral, textSans } from '@guardian/source-foundations';
+import type { FC, ReactNode } from 'react';
+
+// ----- Component ----- //
+
+type Props = {
+	id: string;
+	blockDescription: string;
+	children: ReactNode;
+};
+
+const skipLinkCss = css`
+	${textSans.medium()}
+	height: 40px;
+	left: -100vw;
+	line-height: 30px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	background: ${neutral[100]};
+	display: block;
+	text-align: center;
+	margin: 0;
+	text-decoration: none;
+	color: ${neutral[0]};
+	&:focus,
+	&:active {
+		border: 5px solid ${border.focusHalo};
+		position: static;
+	}
+	&:visited,
+	&:active {
+		color: ${neutral[0]};
+	}
+`;
+
+// Using single template literals for the labels instead of
+// `after {blockDescription}` to prevent JSX rendering
+// an empty comment in the HTML between the static text
+// and the variable. That comment can make screen readers
+// announce the text oddly.
+// https://github.com/guardian/dotcom-rendering/pull/6950#pullrequestreview-1279406318
+
+const InlineSkipToWrapper: FC<Props> = ({
+	id,
```

Comment:
as this is being passed through to the `a`'s href / `p`'s `id`, could we strip out any invalid characters & possibly add a comment? Based on the spec, I think we need to remove any ascii whitespace (https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute)

Reply:
actually I'm not sure a validator is really worth it, but I think a comment on the component might help so it's clear in the consumer where the value will get used